libopenage/game_singletons_info.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015-2016 the openage authors. See copying.md for legal info.
+
+#pragma once
+
+#include <string>
+
+#include "gui/guisys/public/gui_singleton_items_info.h"
+
+namespace openage {
+
+class Engine;
+
+namespace gui {
+
+/**
+ * Accessible during creation of any singleton QML item.
+ */
+struct GameSingletonsInfo : qtsdl::GuiSingletonItemsInfo {
+	GameSingletonsInfo(Engine *engine, const std::string &data_dir)
+		:
+		GuiSingletonItemsInfo{},
+		engine{engine},
+		data_dir{data_dir} {
+	}
```

Comment:
Advantage is that there is no cpp file to add to CMakeLists.txt. I'll make a cpp file.